std/experimental/allocator/package.d

```diff
@@ -573,11 +573,22 @@ unittest
 
 private T[] uninitializedFillDefault(T)(T[] array) nothrow
 {
-    static immutable __gshared T t;
+    T t = T.init;
     fillWithMemcpy(array, t);
     return array;
 }
 
+pure nothrow @nogc
```

Comment:
The attribute style is totally inconsistent throughout the module, I didn't know which to pick, so I did this, which will be consistent with a future PR of mine which adds missing attributes to a lot of the stuff.